MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/EnggDiffraction/EnggDiffractionViewQtGUI.h

```diff
@@ -14,6 +14,7 @@
 #include "ui_EnggDiffractionQtTabSettings.h"
 
 #include <boost/scoped_ptr.hpp>
+#include <Poco/Path.h>
```

Comment:
Just a minor detail I noticed while skimming thought this PR.
I think you don't need to have this include in the `.h` where you can get by with just a forward declaration of the class that you need (Poco/Path). I think that the same applies to the qwt_plot_curve in the next line.
In general it's a good idea not to include headers in header files unless they're needed.
In this case it's not that important, as this header is not used anywhere else than in the "ViewQt" implementation.